vignettes/introduction.Rmd

```diff
@@ -128,16 +127,97 @@ Arguments we use here:
 
 Now we are ready to start training our model:
 
-```{r training, error=TRUE}
+```{r training, error=FALSE, message=FALSE, results="hide"}
 vwtrain(vwmodel = vwmodel, data = train_file_path, passes = 100)
 ```
 
+```{}
```

Comment:
Remove `{}` and just use  ` ```r `

Reply:
Strange, but with ` ```r ` on I get a different color of the text. And with `{}` I get the same color as for regular `R` output

Reply:
Two different things. The first is pandoc only, the seconone is knitr/rmarkdown/pandoc hence more options to change things and set themes.

Reply:
I see I will add this changes

Reply:
I will push this update after we merge [PR#13](https://github.com/rvw-org/rvw/pull/13)